packages/venia-concept/src/components/SignIn/signIn.js

```diff
@@ -85,6 +90,11 @@ class SignIn extends Component {
         this.props.showCreateAccountForm();
     };
 
+    handleForgotPassword = () => {
+        this.props.setDefaultUsername(this.state.username);
+        this.props.onForgotPassword();
```

Comment:
Since we're calling `this.props.onForgotPassword` directly, let's add a `.isRequired` to it in `propTypes`.

Reply:
Yeah, I agree